cpu/esp32/Makefile.dep

```diff
@@ -48,6 +48,14 @@ ifneq (,$(filter esp_idf_nvs_flash,$(USEMODULE)))
     USEMODULE += pthread
 endif
 
+ifneq (,$(filter periph_rtc,$(USEMODULE)))
+    FEATURES_OPTIONAL += esp_rtc_timer_32k
+endif
+
+ifneq (,$(filter esp_rtc_timer_32k,$(FEATURES_USED)))
+    USEMODULE += esp_rtc_timer_32k
+endif
```

Comment:
I'm a bit surprised this works, since FEATURES_USED only gets set by `Makefile.features`, but I guess that's just the magic of `make`s dynamic vatriables.
I compile-tested `esp32-wrover-kit` and there the `esp_rtc_timer_32k` path gets build, as opposed to `esp32-wroom-32` - nice!